querybook/webapp/components/Search/SearchResultItem.scss

```diff
@@ -30,11 +30,23 @@
             .result-items-top {
                 padding-right: var(--padding-sm);
                 align-items: baseline;
+                width: 100%;
 
-                .result-item-title {
+                > div:first-child {
+                    flex-grow: 1;
                     overflow: hidden;
+                    .Text {
+                        width: 100%;
+                        .result-item-title {
+                            overflow-wrap: break-word;
+                        }
+                    }
+                }
+
```

Comment:
for the title, I think you only need
```
.result-item-title {
    overflow-wrap: anywhere
}
```
`break-word` doesn't work on underscores

Reply:
<img width="588" alt="Screenshot 2023-05-12 at 1 46 20 PM" src="https://github.com/pinterest/querybook/assets/29313935/e5ee12ac-bc84-456e-bf25-bea541e692a1">

neither does what you're suggesting. it looks the same as the current implementation